tern/analyze/docker/run.py

```diff
@@ -180,3 +180,26 @@ def execute_dockerfile(args):  # noqa C901,R0912
     if args.name == 'report':
         if not args.keep_wd:
             report.clean_working_dir()
+
+
+def build_multistage(dockerfile_path):
+    """Given a multistage dockerfile, first get the dockerfiles for each
+    stage and then build the dockerfiles.
+    TO DO:
+    1.Add image analyze
+    2. Clean the temp dockerfiles"""
+    dfobj_multi = dockerfile.get_dockerfile_obj(dockerfile_path)
+    file_path_list = dockerfile.get_multistage_image_dockerfiles(dfobj_multi)
+    print(file_path_list)
+    for dfile in file_path_list:
```

Comment:
Same logic with execute_dockerfile(), we are going to loop on this for each dockerfile.